service-base/src/main/java/org/eclipse/hono/service/metric/MicrometerBasedMetrics.java

```diff
@@ -49,6 +50,10 @@
      * The name of the meter for authenticated connections.
      */
     public static final String METER_CONNECTIONS_AUTHENTICATED = "hono.connections.authenticated";
+    /**
+     * This metric is used to keep track of the amount of time that the authenticated devices are connected to Hono.
+     */
+    public static final String METER_CONNECTIONS_AUTHENTICATED_TIME = "hono.connections.authenticated.time";
```

Comment:
IMHO *time* should be replaced with something more specific to the purpose, e.g. *duration*?

Reply:
I will update it.